src/tests/services/actions/permission-mode-action-creators.spec.ts

```diff
@@ -11,10 +11,10 @@ describe('Graph Explorer Permission mode creator\'', () => {
 
     it('creates CHANGE_PERMISSIONS_MODE_SUCCESS when changeMode is called with DISPLAY_APPLICATION_PERMISSIONS', () => {
```

Comment:
That comment should probably be changed to `'creates CHANGE_PERMISSIONS_MODE_SUCCESS when changeMode is called with PERMISSION_MODE_TYPE.TeamsApp'`

```suggestion
    it('creates CHANGE_PERMISSIONS_MODE_SUCCESS when changeMode is called with PERMISSION_MODE_TYPE.TeamsApp', () => {
```

Reply:
Converting your comments to suggestions so you can get credit for them c: